CDCgov/prime-reportstream
# Add abnormal_flag to sender CSV schemas

This PR add abnormal_flag to sender CSV schemas so it can be used in receiver filtering.

Test Steps:
1. Copy settings from attached file for il into local organizations.yml file
2. Run ./prime data command with attached test file for organization with csv-upload-covid-19 schema
3. Expected output from test file:
  - For receiver il-phd.elr-antigen - produce 1 output HL7 message with a positive result for an antigen test
  - For receiver il-phd.elr-molecular - produce 2 output HL7 messages (1 positive, 1 negative) for a molecular test

IL wants to receive all molecular test results +positive results for antigen tests, all other tests should be filtered out.

## Changes
- adds abnormal_flag to 


## Checklist

### Testing
- [ x] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?


### Process
- [X ] Includes a summary of what a code reviewer should test/verify?

## Linked Issues
- Fixes #8269 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #8086
[Il-new filter-settings.txt](https://github.com/CDCgov/prime-reportstream/files/10769913/Il-new.filter-settings.txt)
[test.csv](https://github.com/CDCgov/prime-reportstream/files/10769915/test.csv)
